script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -361,26 +366,26 @@ Then finally, began the autumn of my second year of high school.
 [sha:46a8dc4d84ed6af12630d7a56f8e75e9bfc49374]{
 -- Page 26, Offset 1216.
 -- あの時―――別れ際に先生が言っていたような特別な出来事はまったく起こらなかったし、自分も先生のくれた眼鏡をつけているかぎり『線』を見る事はない。
-The special event that Sensei told me of when we parted never occurred,
+The kind of special incident that Sensei told me about hasn't occurred in the slightest,
```

Comment:
a point event like an incident can't really occur in fractional amounts, so "in the slightest" doesn't work here. I'd just revert that fragment to `me about never occurred`,

Reply:
Good point